src/main/msp/msp.c

```diff
@@ -1151,6 +1151,15 @@ static bool mspProcessOutCommand(mspDescriptor_t srcDesc, int16_t cmdMSP, sbuf_t
         }
         break;
 
+case MSP_NAME:
+        {
+            const int nameLen = strlen(pilotConfig()->craftName);
+            for (int i = 0; i < nameLen; i++) {
+                sbufWriteU8(dst, pilotConfig()->craftName[i]);
+            }
+        }
```

Comment:
```suggestion
        const int nameLen = strlen(pilotConfig()->craftName);
        for (int i = 0; i < nameLen; i++) {
            sbufWriteU8(dst, pilotConfig()->craftName[i]);
        }
```

Reply:
The unnamed scope is unnecessary here

Reply:
Copy pasta from old PR.

Fixed.

Reply:
Looks like it's actually necessary to encapsulate `const int nameLen`, my bad...